Sources/Xcodeproj/pbxproj().swift

```diff
@@ -330,6 +311,34 @@ public func pbxproj(xcodeprojPath: AbsolutePath, graph: PackageGraph, extraDirs:
             // Also add the source file to the compile phase.
             compilePhase.addBuildFile(fileRef: srcFileRef)
         }
+
+        // Add the `include` group for a libary C language target.
+        if case let clangModule as ClangModule = module, clangModule.type == .library, isDirectory(clangModule.includeDir) {
+            let includeDir = clangModule.includeDir
+            let includeGroup = makeGroup(for: includeDir)
+            for header in try walk(includeDir) where header.extension == "h" {
```

Comment:
I think so yes, but it looks like we don't recognise them for modulemap gen either, I'll file a bug to track C++ header support